docs/canarytokens/office365.md

```diff
@@ -0,0 +1,197 @@
+    name: Insert Office365 mail Canarytokens
+    url: /api/v1/canarytoken/office365/create
+    method: POST
+    description: Inserts the Office365 mail Canarytoken into specified email addresses
+    params:
+      - name: auth_token
+        required: true
+        type: string
+        description: A valid auth token
+      - name: flock_id
+        required: false
+        type: string
+        default: "'flock:default'"
+        description: A valid flock_id (defaults to the [Default Flock](/guide/terminology.html#default-flock))
+      - name: user_emails
+        required: true
+        type: string
+        description: A comma separated list of the user email addresses to insert the Canarytoken into.
+      - name: is_exchange
+        required: false
+        type: boolean
+        default: false
+        description: If your email is hosted by a Microsoft Exchange server instead of Office365.
+      - name: exchange_username
+        required: false
+        type: string
+        description: The username used to authenticate to the Office365 exchange.
+      - name: exchange_password
+        required: false
+        type: string
+        description: The password used to authenticate the specified `exchange_username` to the Office365 exchange.
+    response: A JSON structure with a result indicator
+  save_office_365_mail_template:
+    name: Save Office365 mail template
+    url: /api/v1/canarytoken/office365/email
+    method: POST
+    description: Saves the Office365 mail template to be used when tokening Office365 mailboxes
+    params:
+      - name: auth_token
+        required: true
+        type: string
+        description: A valid auth token
+      - name: flock_id
+        required: false
+        type: string
+        default: "'flock:default'"
+        description: A valid flock_id (defaults to the [Default Flock](/guide/terminology.html#default-flock))
+      - name: email_html
+        required: true
+        type: string
+        description: The HTML of the email template
+      - name: email_subject
+        required: true
+        type: string
+        description: The subject of the email
+    response: A JSON structure with the html and subject information with a result indicator
+  get_office_365_mail_template:
+    name: Get Office365 mail template
+    url: /api/v1/canarytoken/office365/email
+    method: GET
+    description: Gets the Office365 mail template to be used when tokening Office365 mailboxes
+    params:
+      - name: auth_token
+        required: true
+        type: string
+        description: A valid auth token
+      - name: flock_id
+        required: false
+        type: string
+        default: "'flock:default'"
+        description: A valid flock_id (defaults to the [Default Flock](/guide/terminology.html#default-flock))
+    response: A JSON structure with the html and subject information with a result indicator
+  office_365_oauth_token:
+    name: Get Office365 oauth token
```

Comment:
How does `Request Office365 OAuth token` sound? "Get" seems to indicate a `get` request and may sound confusing.